source/utilities.tex

```diff
@@ -18337,32 +18337,32 @@
 forms the logical conjunction of its template type arguments.
 
 \pnum
-For a specialization \tcode{conjunction<B1, ..., BN>},
-if there is a template type argument \tcode{Bi} for which \tcode{bool(Bi::value)} is \tcode{false},
-then instantiating \tcode{conjunction<B1, ..., BN>::value}
-does not require the instantiation of \tcode{Bj::value} for \tcode{j > i}.
+For a specialization $\tcode{conjunction<B}_1\tcode{,} \cdots\tcode{, B}_N\tcode{>}$,
+if there is a template type argument $\tcode{B}_i$ for which $\tcode{bool(B}_i\tcode{::value)}$ is \tcode{false},
+then instantiating $\tcode{conjunction<B}_1\tcode{,} \cdots\tcode{, B}_N\tcode{>::value}$
+does not require the instantiation of $\tcode{B}_j\tcode{::value}$ for $j > i$.
```

Comment:
This is somewhat unrelated, but I would feel cosier if we stated the latter range as `$i < j \leq N$`. It just leaves no room for someone saying that $j = 5000$ isn't defined.